config.yml

```diff
@@ -9,12 +9,13 @@ params:
   # Conference info
   Name: "GOCON CANADA"
   Description: "Canadian Go Conference"
-  Date: "May/April 2019"
+  Date: "May 11, 2019 "
   # Price: "UNKNOWN" # If your event is free, just comment this line
-  Venue: "TBA"
-  Address: "1 York street"
+  Venue: "The Cathedral Church of St James"
```

Comment:
I'd recommend changing this to:

> St. James Cathedral Centre

this shows up better on google searches.

Reply:
done